### ISSUE TITLE
Tournament search returns en object instead of a list

### ISSUE THREAD

**Issue by toin0u** [RoyaleAPI/cr-api#275]
If a search has only one result, the API returns the object and not a list with this object. According to the [documentation](http://docs.cr-api.com/#/endpoints/tournaments_search). We might always get a list.

## Request Details:
```
GET https://api.cr-api.com/tournaments/search?name=verilir
```
Please note that when you would like to test the request - the tournament might not be there anymore.

## Response Details:
HTTP Response code: `200`

```
{
	"tag": "U0Q999L",
	"type": "passwordProtected",
	"status": "inProgress",
	"creatorTag": "#8QV2VJ9PY",
	"name": "AA klana gelene şifre verilir",
	"description": "hergün 1 turnuva klan ismi Yarbay",
	"maxCapacity": 50,
	"preparationDuration": 7200,
	"duration": 3600,
	"createTime": 1520416391,
	"startTime": null,
	"endTime": null,
	"playerCount": 14,
	"members": []
}
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there?
It has been always there.

 2. Expected behaviour: 
```
[
  {
	"tag": "U0Q999L",
	"type": "passwordProtected",
	"status": "inProgress",
	"creatorTag": "#8QV2VJ9PY",
	"name": "AA klana gelene şifre verilir",
	"description": "hergün 1 turnuva klan ismi Yarbay",
	"maxCapacity": 50,
	"preparationDuration": 7200,
	"duration": 3600,
	"createTime": 1520416391,
	"startTime": null,
	"endTime": null,
	"playerCount": 14,
	"members": []
  }
]
```

**Comment by fourjr** [comment]
@selfish, is this intended or a bug? If its indented, ill add a note in documentation. 

This might also be occurring for /clan/search

**Comment by selfish** [comment]
Fixed in `v10.6.9`
